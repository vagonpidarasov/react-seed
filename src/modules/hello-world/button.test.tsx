import React from 'react';
import {render} from '@testing-library/react';
import {HelloWorldComponent} from './button';

describe('Button', () => {
    test('matches snapshot with default args', async () => {
        const {container} = render(<HelloWorldComponent/>);
        expect(container).toMatchSnapshot();
    });

    test('matches snapshot with default args', async () => {
        const {container} = render(<HelloWorldComponent/>);
        expect(container.firstChild.textContent).toBe('Click Me');
    });
});
