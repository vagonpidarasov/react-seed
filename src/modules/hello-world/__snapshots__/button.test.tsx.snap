// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches snapshot with default args 1`] = `
<div>
  <div>
    Click Me
  </div>
</div>
`;
